README.md

```diff
@@ -1,6 +1,6 @@
 # IoT Edge
 
-[![Build Status](https://msazure.visualstudio.com/One/_apis/build/status/Custom/Azure/IoT/Edge/Core/Azure-IoT-Edge-Core%20CI?branchName=master)](https://msazure.visualstudio.com/One/_build/latest?definitionId=45137&branchName=master)
+[![Build Status](https://dev.azure.com/msazure/One/_apis/build/status/Custom/Azure/IoT/Edge/Core/Azure-IoT-Edge-Core%20CI?branchName=main)](https://dev.azure.com/msazure/One/_build/latest?definitionId=45137&branchName=main)
```

Comment:
Should this be 1.2 instead?

Reply:
The main purpose of this badge is to give people a quick summary of build status when they open up the project's home page on GitHub (https://github.com/azure/iotedge), which will always display the README for the default branch. We _could_ go update any other branches so their badges point to the branch-appropriate pipeline results, but I don't think it's necessary. And if we were keeping branches in sync via merging rather than cherry-picking (which I believe we should be doing), changes like this would be more hassle than benefit.